mmf/ncdf_animation.c

```diff
@@ -76,7 +76,7 @@ int ncdfCloseAni(long naniVars, PUBVAR **ani_out_vars, int ani_out_flag) {
 
     sprintf(ani_path, "%s.nc", *((char **) control_var("ani_output_file")));
 
-    retval = nc_create(ani_path, NC_CLOBBER, &ncdf);
+    retval = nc_create(ani_path, NC_NETCDF4, &ncdf);
```

Comment:
I know this is merged already, but this is actually changing the behavior of the model too.  I'd suggest changing this to `NC_NETCDF4 | NC_CLOBBER`